lib/gears/timer.lua

```diff
@@ -185,7 +185,6 @@ end
 -- @tparam number timeout Timeout in seconds (e.g. 1.5).
 -- @tparam function callback Function to run.
 -- @treturn timer The timer object that was set up.
--- @see timer.weak_start_new
```

Comment:
But it should work here actually, since it's defined as `timer.weak_start_new` here.